novawallet/Modules/Onboarding/AccountCreate/AccountCreateInteractor.swift

```diff
@@ -30,4 +30,16 @@ extension AccountCreateInteractor: AccountCreateInteractorInputProtocol {
             presenter.didReceiveMnemonicGeneration(error: error)
         }
     }
+
+    func createMetadata() -> MetaAccountCreationMetadata? {
```

Comment:
Why do we need both provideMetadata and createMetadata?

Reply:
In case the user does not tap on the mnemonic card but checks all the boxes, we don't have the metadata to pass the mnemonic to the next screen. Therefore, we use `createMetadata()` to get it synchronously when the user taps the continue button.